utils/coreutils/utils.go

```diff
@@ -400,3 +400,17 @@ func SetCliExecutableName(executableName string) {
 func GetCliExecutableName() string {
 	return cliExecutableName
 }
+
+// Get the explicit list by number of elements.
```

Comment:
Get the explicit list by number of elements.
-->
Turn a list of strings into a sentence.